websockets/README.md

```diff
@@ -0,0 +1,81 @@
+# Websockets samples
+
+The sample takes the application generated in the DBOS quick start[https://docs.dbos.dev/getting-started/quickstart] and add a websocket end point to it.
+
+## Code
+
+### Server
+
+The method sendToClient in src/operations.ts accepts the websocket connection request and continuosly
+sends a random number to the client every second
+
+
+### Client
+
+wsclient.py is the client code that requests a websocket connection. It will stay alive and continue
+to receive data from the server.
+
+
+## Deploying the service
+
+Follow the instructions in the quickstart[https://docs.dbos.dev/getting-started/quickstart] to deploy the app either to the cloud or run locally.
+
+On successfull deployment, you will see a message
+
+2024-08-28 18:53:52 [info]: Access your application at https://yourdomain/ 
```

Comment:
Please format all outputs, commands and the like to come blocks or `backticks`

Reply:
fixed